src/odemis/acq/align/test/clspots_optical_autofocus_test.py

```diff
@@ -44,34 +44,17 @@ class TestAutofocusSim(unittest.TestCase):
     """
     Test simulated auto focus for CL spots.
     """
-    backend_was_running = False
 
     @classmethod
     def setUpClass(cls):
-        try:
-            test.start_backend(CLSPOTS_SIM_CONFIG)
-        except LookupError:
-            logging.debug("A running backend is already found, skipping tests")
-            cls.backend_was_running = True
-            return
-        except IOError as exp:
-            logging.error(str(exp))
-            raise
+        if not TEST_NOHW:
+            raise unittest.SkipTest("No simulator running. Skipping tests.")
```

Comment:
So these tests need the simulator to run. But it should still be fine to run the simulator backend if the computer is connected to the real hardware? So never skip, just always run the simulator.